app/src/main/res/values/strings.xml

```diff
@@ -377,6 +377,7 @@
     <string name="importer_help_authenticator_plus">Supply an Authenticator Plus export file obtained through <b>Settings -> Backup &amp; Restore -> Export as Text and HTML</b>.</string>
     <string name="importer_help_authy">Supply a copy of <b>/data/data/com.authy.authy/shared_prefs/com.authy.storage.tokens.authenticator.xml</b>, located in the internal storage directory of Authy.</string>
     <string name="importer_help_andotp">Supply an andOTP export/backup file.</string>
+    <string name="importer_help_bitwarden">Supply a Bitwarden export/backup file.</string>
```

Comment:
Could you add a note at the end that says: "Encrypted files are not supported".?